src/FINDOLOGIC/Api/Config.php

```diff
@@ -24,17 +24,39 @@ class Config
     private $serviceId;
 
     /** @var string */
-    private $apiUrl;
+    private $apiUrl = self::DEFAULT_TEMPLATE_API_URL;
 
     /** @var float */
-    private $alivetestTimeout;
+    private $alivetestTimeout = self::DEFAULT_ALIVETEST_TIMEOUT;
 
     /** @var float */
-    private $requestTimeout;
+    private $requestTimeout = self::DEFAULT_REQUEST_TIMEOUT;
 
     /** @var Client */
     private $httpClient;
 
+    /**
+     * Sets a specified config value and validates them according to the given validation rules.
+     *
+     * @param string $key
+     * @param mixed $value
+     * @param array $validationRules
+     */
+    private function setConfigValue($key, $value, array $validationRules)
```

Comment:
Great idea for a helper! :+1: Hope I am praising you not myself :stuck_out_tongue_winking_eye:

Reply:
Yeah, but good idea !!